examples/211-dotnet-WebClient-Intent-Detection/README.md

```diff
@@ -0,0 +1,34 @@
+## Example: using the service with \<MemoryWebClient>
```

Comment:
pls update the description and the cod. This example shows how to use KM for intent detection... etc. using... etc.

Reply:
done